app/client/components/delivery/address/deliveryAddressForm.tsx

```diff
@@ -411,133 +469,211 @@ const Form = (props: FormProps) => {
+              font-weight: bold;
+              line-height: 48px;
+              background-color: ${palette.neutral["97"]};
+              border-bottom: 1px solid ${palette.neutral["86"]};
             `}
           >
-            {props.warning}
-          </div>
+            Delivery address and instructions
+          </legend>
+          <Input
+            label={"Address line 1"}
+            width={30}
+            value={props.addressLine1}
+            changeSetState={props.setAddressLine1}
+            inErrorState={
+              props.formStatus === formStates.VALIDATION_ERROR &&
+              !props.formErrors.addressLine1?.isValid
+            }
+            errorMessage={props.formErrors.addressLine1?.message}
+          />
+          <Input
+            label="Address line 2"
+            width={30}
+            value={props.addressLine2}
+            changeSetState={props.setAddressLine2}
+            optional={true}
+          />
+          <Input
+            label="Town or City"
+            width={30}
+            value={props.town}
+            changeSetState={props.setTown}
+            inErrorState={
+              props.formStatus === formStates.VALIDATION_ERROR &&
+              !props.formErrors.town?.isValid
+            }
+            errorMessage={props.formErrors.town?.message}
+          />
+          <Input
+            label="County or State"
+            width={30}
+            value={props.region}
+            optional={true}
+            changeSetState={props.setRegion}
+          />
+          <Input
+            label="Postcode/Zipcode"
+            width={11}
+            value={props.postcode}
+            changeSetState={props.setPostcode}
+            inErrorState={
+              props.formStatus === formStates.VALIDATION_ERROR &&
+              !props.formErrors.postcode?.isValid
+            }
+            errorMessage={props.formErrors.postcode?.message}
+          />
+          <Select
+            label={"Country"}
+            options={COUNTRIES}
+            width={30}
+            additionalCSS={css`
+              margin-top: 14px;
+            `}
+            value={props.country}
+            changeSetState={props.setCountry}
+            inErrorState={
+              props.formStatus === formStates.VALIDATION_ERROR &&
+              !props.formErrors.country?.isValid
+            }
+            errorMessage={props.formErrors.country?.message}
+          />
+          {props.enableDeilveryInstructions && (
+            <label
+              css={css`
+                display: block;
+                color: ${palette.neutral["7"]};
+                ${textSans.medium()};
+                font-weight: bold;
+              `}
+            >
+              Instructions
+              <div>
+                <textarea
+                  id="delivery-instructions"
+                  name="instructions"
+                  rows={2}
+                  maxLength={500}
```

Comment:
not sure your latest changes are pushed up?
```suggestion
                  maxLength={240}
```